.github/workflows/dev.yml

```diff
@@ -124,7 +124,7 @@ jobs:
         shell: bash
         run: |
           gem install test-unit
-          pip install "cython>=3" setuptools pytest requests setuptools-scm
+          pip install "cython>=3" meson-python "cmake<4" pytest requests meson-python setuptools-scm
```

Comment:
It looks like cmake=4.0.0 was just released today and is causing some issues with mimalloc and boost resolution. Hope that this upper bound won't have to stay long